packages/frontend/src/components/ChartDownload.tsx

```diff
@@ -180,6 +191,21 @@ const ChartDownloadOptions: React.FC<DownloadOptions> = ({
                 }))}
             />
 
+            {type === DownloadType.PNG && (
+                <Select
+                    size="xs"
+                    id="background-transparency"
+                    value={backgroundTransparent ? 'Transparent' : 'Opaque'}
+                    onChange={(value) =>
+                        setBackgroundTransparent(value === 'Transparent')
+                    }
+                    data={[
+                        { value: 'Transparent', label: 'Transparent' },
+                        { value: 'Opaque', label: 'Opaque' },
+                    ]}
+                />
+            )}
+
```

Comment:
https://v6.mantine.dev/core/segmented-control/

Could we use a `Segmented Control` where `opaque` is the default? 

Just made the change to the original ticket 👍